rotkehlchen/tests/unit/solana_decoders/test_spl_transfers.py

```diff
@@ -51,7 +51,7 @@ def test_spl_token_transfer(
         address=SolanaAddress('GabwTHe1P9wLKCqqBNBnAPgD71K8j8Dp4Uk4L6EyFUnK'),
     )]
     with solana_inquirer.database.conn.read_ctx() as cursor:
-        assert cursor.execute('SELECT * FROM key_value_cache').fetchall() == [
+        assert cursor.execute('SELECT * FROM key_value_cache WHERE name LIKE "solana%"').fetchall() == [  # noqa: E501
```

Comment:
why this change in this PR?

Reply:
the scope of the db fixture so it contained entries from another test